db/schema.rb

```diff
@@ -57,7 +47,7 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
-  create_table "active_storage_variant_records", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+  create_table "active_storage_variant_records", charset: "utf8mb4", force: :cascade do |t|
```

Comment:
Locally I have collation set in my datbase.yml to `utf8mb4_unicode_ci` in efforts to resolve https://github.com/sudara/alonetone/issues/732 — so this change is desirable.